AndelaOSP/art-dashboard
# #165785610 update andela centers functionality

## What does this PR do?
-  Updates Andela centers functionality
## Description of Task to be completed?
- Removes add Andela Center Button
## How should this be manually tested?
- Run the app
- Click on Andela Centres
- A list of Andela Centres is displayed without an `Add Centre` button

## What are the relevant pivotal tracker stories?
[#165785610](https://www.pivotaltracker.com/story/show/165785610)

## Any background context you want to add?
`N/A`
## Important notes
`N/A`
## Packages installed
`N/A`

## Deployment note

## Related PRs branch | PR (branch_name) | (pr_link)
Branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()
## Todos
## Screenshots

<img width="1310" alt="Screenshot 2019-05-09 at 12 37 01" src="https://user-images.githubusercontent.com/31400129/57449354-5501fb00-7264-11e9-9a44-dcfcd823d6d9.png">
